ccp/lib/utils/command.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from ccp.lib.exceptions import CommandOutputError
+import subprocess
+
+
+def run_command(cmd, shell=False):
+    """
+    Runs a shell command
+    :param cmd: The command that needs to be run.
+    :param shell: Default False: Whether to run raw shell commands with '|' and
+    redirections
+    :return: Command output
+    :raises subprocess.CalledProcessError
+    :raises ccp.lib.exceptions.CommandOutputError
+    """
+    p = subprocess.Popen(cmd, shell=shell)
+    out, err = p.communicate()
+    if err:
+        raise CommandOutputError(err)
```

Comment:
Let the caller decide what needs to be done if `err` comes - and let the caller handle the exceptions.

Also, this breaks the functionality where `run_cmd` and `run_cmd2` are used, expecting `1` and `2` return values respectively.

Reply:
Done